data_masks/redactor.go

```diff
@@ -36,9 +36,8 @@ type Redactor struct {
 	// userConfig should exit.
 	exitChannel chan struct{}
 
-	// Whether this redactor is still running. False if and only if exitChannel is
-	// closed.
-	running *atomic.Bool
+	// Ensures that exitChannel is closed at most once.
+	closeExitChannelOnce *sync.Once
```

Comment:
There's no need for a * here, but it's fine.

Reply:
The `*` avoids copying the `Once` instance when a `Redactor` is copied.

Reply:
But I suppose `Redactor` instances shouldn't be copied regardless!

Reply:
Not with that channel sitting there ;)